defaults/parameters.yaml

```diff
@@ -65,7 +65,8 @@ files:
   auspice_config: "defaults/auspice_config.json"
   lat_longs: "defaults/lat_longs.tsv"
   description: "defaults/description.md"
-  clades: "defaults/clades.tsv"
+  clades: "defaults/clades_nextstrain.tsv"
```

Comment:
I wonder if we should keep the default file named as `clades.tsv`, since this is what we refer to in the docs (e.g., [labeling clades](https://docs.nextstrain.org/projects/ncov/en/latest/guides/workflow-config-file.html#labeling-clades), [the workflow reference guide](https://docs.nextstrain.org/projects/ncov/en/latest/reference/workflow-config-file.html#clades), etc.) and call the derived file for the workflow something different?

If we do rename the default, we just need to update references to it in the docs.

Reply:
Not just, as I use the exact file from another repo ;)

Reply:
I had named it in analogy to `clades_who`, but the latter isn't really part of this workflow. I am fine with renaming it.